apps/linode-marketplace-filecloud/roles/solr/tasks/main.yml

```diff
@@ -0,0 +1,17 @@
+# Solr installation
+- name: installing solr
+  shell:
+    cmd: filecloudcp -s
+
+- name: deleting fccore
+  file: 
+    path: /opt/solrfcdata/var/solr/data/fccore
+    state: absent
+
+- name: restarting solr
+  shell:
+    cmd: service solr restart
```

Comment:
Needs to be redone. Use the `service` or `systemd_service` module to restart the service.